portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,70 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.time.format.DateTimeFormatter;  
+import java.time.ZonedDateTime;   
+import java.time.ZoneId;   
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    final private ArrayList<Comment> allComments = new ArrayList<Comment>();
```

Comment:
nit: the standard order in java is private, then final, ie: "private final ArrayList..."

Reply:
Fixed.